### ISSUE TITLE
Cannot apply transformations in DIODE

### ISSUE THREAD

**Issue by Gabbeo** [spcl/dace#282]
**Describe the bug**
Cannot apply more than one transformation on SDFG in DIODE.
Applied transformations do not show up in the History tab.

**To Reproduce**
Load the following code into DIODE while on commit [https://github.com/spcl/dace/commit/68b667ff107f49631dbbf550be3b5826f9478353](https://github.com/spcl/dace/commit/68b667ff107f49631dbbf550be3b5826f9478353) and try to apply MapReduceFusion and MapTiling.

```#!/usr/bin/env python
from __future__ import print_function

import argparse
import dace
import numpy as np

M = dace.symbol('M')
K = dace.symbol('K')
N = dace.symbol('N')

@dace.program(dace.float64[M, K], dace.float64[K, N], dace.float64[M, N])
def gemm(A, B, C):
    tmp = dace.define_local([M, N, K], dtype=A.dtype)

    @dace.map(_[0:M, 0:N, 0:K])
    def multiplication(i, j, k):
        a << A[i, k]
        b << B[k, j]
        out >> tmp[i, j, k]

        out = a * b

    dace.reduce(lambda a, b: a + b, tmp, C, axis=2, identity=0)


if __name__ == "__main__":
    n = 64
    M.set(n)
    K.set(n)
    N.set(n)

    A = np.random.rand(M.get(), K.get()).astype(np.float64)
    B = np.random.rand(K.get(), N.get()).astype(np.float64)
    C = np.zeros([M.get(), N.get()], dtype=np.float64)
    C_regression = np.zeros_like(C)

    gemm(A, B, C)
```

**Comment by tbennun** [comment]
This bug was introduced recently (not sure which commit caused it), it's great to have an issue for it. Thanks.

**Comment by tbennun** [comment]
Fixed in #376.
